src/recipes/build.ts

```diff
@@ -0,0 +1,150 @@
+import { CycliToolbox, Mode, Platform, ProjectContext } from '../types'
+import { join } from 'path'
+
+const createBuildWorkflowForAndroid = async (
+  toolbox: CycliToolbox,
+  context: ProjectContext,
+  { mode, expo }: { mode: Mode; expo: boolean },
+  workflowProps: Record<string, string> = {}
+): Promise<string> => {
+  const gradleCommands =
+    mode === 'debug' ? 'assembleDebug' : 'assembleRelease assembleAndroidTest'
+
+  let script = [
+    'cd android &&',
+    `./gradlew ${gradleCommands}`,
+    `-DtestBuildType=${mode}`,
+    '-Dorg.gradle.jvmargs=-Xmx4g',
+  ].join(' ')
+
+  if (expo) {
+    script = `npx expo prebuild --${context.packageManager} && ${script}`
+  }
+
+  await toolbox.scripts.add(`build:${mode}:android`, script)
+
+  const workflowFileName = await toolbox.workflows.generate(
+    join(`build-${mode}`, `build-${mode}-android.ejf`),
+    context,
+    workflowProps
+  )
+
+  toolbox.interactive.success(`Created Android ${mode} build workflow.`)
+
+  return workflowFileName
+}
+
+const createBuildWorkflowForIOs = async (
```

Comment:
IOS? :D

Reply:
changed in ef4d52054da13215e5dbe33ba57e407e5fe26939